dcm4che-core/src/main/java/org/dcm4che3/data/Fragments.java

```diff
@@ -256,4 +256,21 @@ private int itemHashCode(Object e) {
         }
     }
 
+
+
+    /**
+     * Determine the length of an object found in the BulkData
```

Comment:
You mean the length of an item in the Fragments? For consistency with the other methods it should be called `itemLength(Object e)`